fed/mosaic-application/src/main/java/org/eclipse/mosaic/fed/application/ambassador/simulation/communication/AbstractCommunicationModule.java

```diff
@@ -142,7 +144,10 @@ void logEnableConfigurationNull() {
      */
     protected Integer sendCam(MessageRouting routing) {
         final CamBuilder camBuilder = owner.assembleCamMessage(new CamBuilder());
-        final Cam cam = new Cam(routing, camBuilder.create(owner.getSimulationTime(), owner.getId()));
+        final Cam cam = new Cam(routing,
+                camBuilder.create(owner.getSimulationTime(), owner.getId()),
+                ObjectUtils.defaultIfNull(configuration.camPayloadMinimalLength, 200L)
```

Comment:
We could consider adding a constant for the `200L`

Reply:
done